### PR TITLE
Add wdt-emoji-bundle@0.1.4 w/ git auto-update

### PR DESCRIPTION
**Pull request by hare1039**
Pull request for issue: #9911 
Related issue(s): # #

@cdnjs/intern3 Please help me review this PR. Thanks.
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/needim/wdt-emoji-bundle/
 * Official website (optional, not the repository): http://ned.im/wdt-emoji-bundle
 * NPM package url (optional): https://www.npmjs.com/package/wdt-emoji-bundle
 * License and its reference: MIT, [ref](https://github.com/needim/wdt-emoji-bundle/blob/master/LICENSE.txt)
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 13
   - Count of stars: 235
   - Count of forks: 36
 * NPM download stats (optional):
   - Downloads in the last day: 3
   - Downloads in the last week: 10
   - Downloads in the last month: 175

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/wdt-emoji-bundle/0.1.4/wdt-emoji-bundle.js [hunk 1]
```diff
@@ -0,0 +1,952 @@
...
+    },
+    skinColor    : 'skin-1',
+    allowNative  : false,
+    emojiType    : 'apple',
+    emojiSheets  : {
+      'apple'   : '/sheets/sheet_apple_64_indexed_128.png',
+      'google'  : '/sheets/sheet_google_64_indexed_128.png',
+      'twitter' : '/sheets/sheet_twitter_64_indexed_128.png',
+      'emojione': '/sheets/sheet_emojione_64_indexed_128.png'
+    },
```
Comment (sufuf3) [inline ajax/libs/wdt-emoji-bundle/0.1.4/wdt-emoji-bundle.js]: I think you have to include `sheets/*`

Comment (hare1039) [inline ajax/libs/wdt-emoji-bundle/0.1.4/wdt-emoji-bundle.js]: @sufuf3 I add the files. Please review this PR again. Thanks.

**Comment by hare1039** [comment]
@pvnr0082t Please help me on reviewing this PR. Thanks.

File: ajax/libs/wdt-emoji-bundle/package.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/needim/wdt-emoji-bundle.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "wdt-emoji-bundle.@(*js|css)",
+          "emoji.min.js",
```
Comment (PeterDaveHello) [inline ajax/libs/wdt-emoji-bundle/package.json]: Can we combine the above two rules?

**Comment by hare1039** [comment]
@pvnr0082t I fix the tag and combine the rules. Please review this PR again. Thanks.

File: ajax/libs/wdt-emoji-bundle/package.json [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    "url": "git://github.com/needim/wdt-emoji-bundle.git"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/needim/wdt-emoji-bundle.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "*emoji*.@(*js|css)",
```
Comment (pvnr0082t) [inline ajax/libs/wdt-emoji-bundle/package.json]: We can improve it to `"*emoji*.+(js|css|map)"`

Comment (hare1039) [inline ajax/libs/wdt-emoji-bundle/package.json]: @pvnr0082t I improved the rules. Please review this PR again. Thanks.

**Pull request merged**